core/misc/ajax.js

```diff
@@ -673,6 +673,26 @@ Backdrop.ajax.prototype.commands = {
   },
 
   /**
+   * Command to add css.
+   *
+   * Uses the proprietary addImport method if available as browsers which
+   * support that method ignore @import statements in dynamically added
+   * stylesheets.
+   */
+  add_css: function (ajax, response, status) {
```

Comment:
This command isn't consistent with our coding standards and the other AJAX commands. It should be camelCase, `addCss()`.